package.json

```diff
@@ -24,7 +24,8 @@
     "build:css": "postcss -o ./src/stylesheets/styles.css -c ./scripts/postcss.config.js ./src/stylesheets/styles.pcss",
     "build:css:min": "cross-env NODE_ENV=production npm run build:css",
     "build:js": "rollup -c",
-    "build": "npm run build:svg && npm run build:css:min && npm run build:js",
+    "build:public": "rimraf public/* && cpy dummy/* public && cpy dist/uploadcare.full.js public",
```

Comment:
We can update the path in 'dummy' examples and that's all. 
./uploadcare.full.js => ./dist/uploadcare.full.js

`build:public` not working without `build:js` 
We don't need another folder with a copy process. It looks hard and doesn't take profit.

Reply:
dev build won't work then, but that's a good point, I'll do it better